mobile/src/screens/BookScreen.tsx

```diff
@@ -106,6 +106,16 @@ export const BookScreen: React.FC<BookScreenProps> = ({ route, navigation }) =>
       setFavorited(false);
     })();
   };
+  // increments click on book analytics
+  // Check in profile screen if data collection is allowed
+  useEffect(() => {
+    (async () => {
+      const allow = await AsyncStorage.getItem("allowAnalytics");
+      if (allow) {
+        await client.incrementClicks(book.id);
+      }
```

Comment:
This needs a try/catch - we don't want the app to crash if the user is offline. No need for anything in the catch block though.